apiserver/facades/controller/sshserver/facade.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package sshserver
+
+import (
+	"github.com/juju/juju/apiserver/common"
+	"github.com/juju/juju/apiserver/facade"
+	"github.com/juju/juju/rpc/params"
+	"github.com/juju/juju/state"
+	"github.com/juju/juju/state/watcher"
+)
+
+// State provides required state for the Facade.
+type State interface {
+	WatchControllerConfig() state.NotifyWatcher
+	SSHServerHostKey() (string, error)
+}
+
+// Facade allows model config manager clients to watch controller config changes and fetch controller config.
+type Facade struct {
+	resources facade.Resources
+
+	ctrlState           State
+	controllerConfigAPI *common.ControllerConfigAPI
+}
+
+func NewFacade(ctx facade.Context, systemState *state.State) *Facade {
```

Comment:
Comment public artefacts

Also, don't have the systemState param be a `*state.State`. Use an interface that can be mocked for testing. And call the param `backend` or something, not `systemState`

Reply:
Done!